docs/architecture/tablets.rst

```diff
@@ -0,0 +1,124 @@
+the same node to balance the shards or across the nodes to balance the global
+load in the cluster.
+
+As a table grows, each tablet can split into two, creating a new tablet.
+The load balancer can migrate the split halves independently to different nodes
+or shards.
+
+The load-balancing process takes place in the background and is performed
+without any service interruption.
+
+Scaling Out
+=============
+
+A tablet can be dynamically migrated to either an existing node or a new node – on
+bootstrap. Paired with consistent topology updates with Raft, tablets allow
+you to add multiple nodes simultaneously. Existing nodes stream data to the new
+ones, and the system load eventually converges to an even distribution as 
+the process completes. 
+
+With tablets, streaming data to a new node is fast and efficient, because:
+
+* Entire SStables are streamed, so less data is read at the source.
+* Cleanup is performed automatically per tablet. Manual cleanup is not required, 
+  making tablets-based streaming user-independent and safer.
+
+  In addition, tablet cleanup is light-weight and efficient, as it doesn't
+  involve rewriting SStables on the existing nodes. This dramatically reduces 
+  the impact of cleanup on the performance of user queries.
+
+The following diagrams show migrating tablets from heavily loaded nodes A and B
+to a new node.
+
+.. image:: images/tablets-load-balancing.png
+
+.. _tablets-enable-tablets: 
+
+Enabling Tablets
+-------------------
+
+Tablets are enabled or disabled on the keyspace level. When you create a new
+keyspace, tablets are enabled by default. You can create a keyspace with tablets
+disabled with the ``tablets_enabled:false`` option:
+
+.. code:: cql
+
+    CREATE KEYSPACE my_keyspace
+    WITH replication = {
+        'class': 'NetworkTopologyStrategy',
+        'replication_factor': 3,
+    } AND tablets = {
+        'enabled': false
+    };
+
+.. warning::
+
+    You cannot ALTER a keyspace to enable or disable tablets.
+    The only way to update the tablet support for a keyspace is to DROP it
+    (losing the schema and data) and then recreate it after redefining 
+    the keyspace schema with ``tablets = { 'enabled': false }`` or 
+    ``tablets = { 'enabled': true }``.
+
+Limitations and Unsupported Features
+--------------------------------------
+
+The following ScyllaDB features are not supported if a keyspace has tablets
+enabled:
+
+* Counters
+* Change Data Capture (CDC)
+* Lightweight Transactions (LWT)
+* Alternator (as it uses LWT)
+
+If you plan to use any of the above features, CREATE your keyspace
+:ref:`with tablets disabled <tablets-enable-tablets>`.
+
+Resharding in keyspaces with tablets enabled has the following limitations:
+
+* ScyllaDB does not support scaling down the number of shards when tablets are
+  allocated to shards that do not exist after backup.
+* ScyllaDB does not reshard data on node restart. Tablets-based tables remain on restart
```

Comment:
> Tablets-based tables remain on restart

It's unclear what you mean, remain where? Maybe: "Tablet replicas remain allocated on the old shards on restart".